tests/test_register.py

```diff
@@ -12,7 +12,7 @@
 
 @pytest.fixture()
 def register_settings(make_settings):
-    """Returns a factory function for settings.Settings for register"""
+    """settings.Settings to use while registering a package"""
```

Comment:
The first word must be a verb per PEP 257. "Create settings.Settings for testing package registration" for example

Reply:
Fixed in latest commit.